<launch>
  <arg name="rvizconfig" default="$(find mxxx)/rviz/laser.rviz" />
  
  <node pkg="mxxx" name="mxxx" type="MXXX_ros_node" output="screen" respawn="true">
    <param name="host_ip" value="192.168.1.201" />
    <param name="port" value="2112" />

    <param name="scan_range_min" value="0" />
    <param name="scan_range_max" value="100" />

    <param name="angle_min" value="-2.356" />
    <param name="angle_max" value="2.356" />

    <param name="frame" value="laser" />
  </node>
  
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

</launch>
